nbcelltesting/main.js

```diff
@@ -395,8 +402,13 @@ define([
 
 
     events.on('preset_activated.CellToolbar', function(event, preset) {
+        create_global_result(preset.name === preset_name);
+    });
+
+
+    var create_global_result = function(show=true) {
```

Comment:
Defining default arguments for function values like you do here is part of ECMAscript 6 and thus not yet supported on all browsers (see https://kangax.github.io/compat-table/es6/, first row).

If you can live with excluding IE/Edge (<14)-Users then I'm okay with this. Otherwise you should do something like

`show = typeof a !== 'undefined' ? a : true;`

to ensure compatibility.